src/llm/llm_calculator.proto

```diff
@@ -26,4 +26,14 @@ message LLMCalculatorOptions {
     optional LLMCalculatorOptions ext = 113473750;
     }
     required string models_path = 1;
+
+    optional uint64 max_num_batched_tokens = 2 [default = 256];
+
+    optional uint64 num_kv_blocks = 3 [default = 364];

```

Comment:
As we update CB library version, let's change that to `cache_size` (not just the name, there's a new parameter in the CB library and we should use it since it's more user friendly).